code/modules/projectiles/guns/energy/tag.dm

```diff
@@ -72,9 +76,16 @@
     else
         makeTaser(user)
 
-/obj/item/weapon/gun/energy/tag/special_check(var/mob/living/carbon/human/M)
-	if(ishuman(M))
-		if(istype(M.wear_suit, needed_vest))
+/obj/item/weapon/gun/energy/tag/special_check(var/mob/living/M)
+	var/suit
+	if(ishuman(M) || ishologram(M))
```

Comment:
Changelog's still wrong at this point unless you're not done yet. This check will not pass for monkeys or nymphs.

Reply:
I know. As per my last comment, I plan to remake laser tag vests so that they can be used for all mobs.